utils/locationUtil.js

```diff
@@ -0,0 +1,80 @@
+const axios = require("axios");
+const getLocationByPostalCode = async (postalCode) => {
+  const url =
+    "https://maps.googleapis.com/maps/api/geocode/json?address=" +
+    postalCode +
+    "&key=" +
+    process.env.GOOGLE_MAPS_API_ACCESS_KEY;
+
+  try {
+    const response = await axios.get(url);
+
+    const body = response.data;
```

Comment:
Need to include a handle for if 0 results are found:


![image](https://user-images.githubusercontent.com/39810669/89095085-2b5c5480-d37f-11ea-8804-0d3bbed55624.png)

(ex. input `L7E 9H2` as a postal code)
![image](https://user-images.githubusercontent.com/39810669/89095102-5050c780-d37f-11ea-8b6d-141eb6cdb7c8.png)